easybuild/easyblocks/o/opencv.py

```diff
@@ -128,6 +135,8 @@ def configure_step(self):
             shlib_ext = get_shared_lib_ext()
             if dep == 'zlib':
                 lib_file = 'libz.%s' % shlib_ext
+            elif dep == 'OpenEXR':
```

Comment:
@lexming Isn't the alternative way via `-DOPENEXR_ROOT` a better approach?

Reply:
Actually, it seems like that's required, since with the current changes I still see this in the output of `cmake`:

```
OpenEXR:                     build (ver 2.3.0)
```

Reply:
ah yep, you're right, just setting `libIex` is not enough it checks for all the libraries in OpenEXR. I took your code in https://github.com/easybuilders/easybuild-easyblocks/pull/2346/commits/4d02973e325d9ae008676c73df2e0d4ec7d2a88e. Thanks!